src/cli/project/platform/remove.rs

```diff
@@ -0,0 +1,84 @@
+use crate::environment::update_prefix;
+use crate::lock_file::{load_lock_file, update_lock_file};
+use crate::prefix::Prefix;
+use crate::Project;
+use clap::Parser;
+use itertools::Itertools;
+use miette::IntoDiagnostic;
+use rattler_conda_types::Platform;
+use std::str::FromStr;
+
+/// Remove a platform(s) to the project file and updates the lockfile.
+#[derive(Parser, Debug, Default)]
+pub struct Args {
+    /// The platform name(s) to add.
+    #[clap(required = true, num_args=1..)]
+    pub platform: Vec<String>,
+
+    /// Don't update the environment, only remove the channel(s) to the lock-file.
+    #[clap(long)]
+    pub no_install: bool,
+}
+
+pub async fn execute(mut project: Project, args: Args) -> miette::Result<()> {
+    // Determine which platforms are missing
```

Comment:
I'm confused by this line :sweat_smile:

Reply:
Just me copy/pasting and being lazy xD

Changed to `// Determine which platforms to remove`